scripts/bootstrap-prefix.sh

```diff
@@ -2036,11 +2069,13 @@ bootstrap_stage2() {
 
 	emerge_pkgs --nodeps "${pkgs[@]}" || return 1
 
-	# Debian multiarch supported by RAP needs ld to support sysroot.
-	for pkg in ${linker} ; do
-		EXTRA_ECONF=$(rapx --with-sysroot=/) \
-		emerge_pkgs --nodeps "${pkg}" || return 1
-	done
+	if [[ ${CHOST}:${DARWIN_USE_GCC} != *-darwin*:0 ]] ; then
```

Comment:
So you just don't want to merge the linker here for clang target?  Can it be set to empty or something?